tests/helpers.py

```diff
@@ -183,9 +183,46 @@ def load_baking_time(_builder: Builder) -> float:
 
 
 class SingleLookupCreator(ColumnCreator):
+    pass
+
+
+class OrderedColumnsLookupCreator(Component):
     @property
-    def standard_lookup_tables(self) -> List[str]:
```

Comment:
it didn't seem to be using this?

Reply:
I don't remember what this was for but Rajan and I had tried to add some feature like this but I think this is deprecated now.